cmd/dist/main.go

```diff
@@ -93,3 +88,24 @@ distribution tool
 		os.Exit(1)
 	}
 }
+
+var imageCommand = cli.Command{
+	Name:  "image",
+	Usage: "image management",
+	Subcommands: cli.Commands{
+		listCommand,
```

Comment:
Actually, the list command is for content, not images. The imagesCommand needs to be renamed to the imagesListCommand.